core/src/marker/markerManager.cpp

```diff
@@ -63,6 +64,19 @@ bool MarkerManager::setStyling(MarkerID markerID, const char* styling) {
     return true;
 }
 
+bool MarkerManager::setBitmap(MarkerID markerID, int width, int height, const unsigned int* bitmapData) {
```

Comment:
Yeah, being able to use compressed image formats would be very useful. The design of that interface will, I think, depend on the use cases that we find in the platform SDKs. For instance, should this function take an array of data or a path to a file? One is simpler but the other allows us to cache repeated images, et cetera. I think this is best left as a follow-up task.